ExtendFinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -20,4 +51,155 @@ public function setProperties()
     {
         parent::setProperties();
     }
+
+    /*
+     * Direct Integration only:
+     * Example on how to add a property with all variants as a JSON.
+     * Each variant includes the corresponding URL and assigned images.
+     *
+     * Example:
+     * {
+     *   "red": {
+     *     "productUrl": "https://www.example.com/de/jacket?number=SW10000-01",
+     *     "images": [
+     *       "https://www.example.com/media/image/73/bc/ef/jacket-red-01.jpg",
+     *       "https://www.example.com/media/image/73/bc/ef/jacket-red-02.jpg",
+     *     ],
+     *   },
+     *   "black": {
+     *     "productUrl": "https://www.example.com/de/jacket?number=SW10000-02",
+     *     "images": [
```

Comment:
Since there is also now `thumbnails`, please add them here to the example as well.